### ISSUE TITLE
Exit from fullscreen with Back button doesnt exit at once in landscape

### ISSUE THREAD

**Issue by luckkmaxx** [libre-tube/LibreTube#4187]
### Steps to reproduce

With a video in landscape fullscreen,
- Press back (phone navigation)
- The video gets zoomed,  moves down a bit, and the statusbar keeps on top.
  - Here the ui controls are out of the screen in the bottom
- Press back again and finally the video is minimized.

### Expected behavior

minimize immediatelly when you press back button

### Actual behavior

described

### LibreTube version

0.16.0 (and older)

### Android version

13

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by Qaz-6** [comment]
You didn't mention details because this problem occurs when auto full screen option is on.

**Comment by Bnyro** [comment]
Could you please test if that's still reproducible in the latest debug builds (specifically after #4607)?

**Comment by luckkmaxx** [comment]
> This problem only occurs if the auto full screen option is turned on

Yes I can confirm this too.

**Comment by Bnyro** [comment]
Fixed with latest release.
